event-schema/event_schema.go

```diff
@@ -349,6 +364,8 @@ func (manager *EventSchemaManagerT) handleEvent(writeKey string, event EventT) {
 			}
 			stats.NewTaggedStat("reload_offloaded_event_model", stats.CountType, stats.Tags{"module": "event_schemas", "writeKey": eventModel.WriteKey, "eventIdentifier": eventModel.EventIdentifier}).Increment()
 		} else if wasArchived {
+			// If we saw event from an archived event model, reload the model into memory
+			// and archive the oldest model then. TODO: A test case for this ?
```

Comment:
`Query`: With regards to the line `374` why would we have to create the model again? We populate from the Db right so we should be able to reload?

Reply:
Good point. I'll dig up a bit to see why that's the case.

Reply:
What's the case then?